pkg/btree/bptree_test.go

```diff
@@ -247,6 +263,57 @@ func (s *StubDataParser) Parse(value []byte) []byte {
 	return []byte{1, 2, 3, 4, 5, 6, 7, 8}
 }
 
+func TestBinarySearchReferencedValues(t *testing.T) {
```

Comment:
This test isn't really testing any of our code since it calls `slices.BinarySearchFunc`. It might be useful to leave as an example but I don't think we should run it with unit tests. Name it `func Example*()` if you want to leave it in: https://go.dev/blog/examples